includes/class-widget.php

```diff
@@ -66,16 +66,29 @@ public function widget( $args, $instance ) {
 		}
 
 		echo '<div class="license">';
-			echo sprintf(
-				'<p><button ' . ( $is_amp ? 'on="tap:republication-tracker-tool-modal"' : '' ) . ' name="%1$s" id="cc-btn" class="republication-tracker-tool-button">%1$s</button></p>',
-				esc_html__( 'Republish This Story', 'republication-tracker-tool' )
-			);
+
+			if ( empty( $instance['layout'] ) || 'modal' === $instance['layout'] ) {
+				echo sprintf(
+					'<p><button ' . ( $is_amp ? 'on="tap:republication-tracker-tool-modal"' : '' ) . ' name="%1$s" id="cc-btn" class="republication-tracker-tool-button modal">%1$s</button></p>',
+					esc_html__( 'Republish This Story', 'republication-tracker-tool' )
+				);
+			}
+
+			if ( 'page' === $instance['layout'] ) {
+				echo sprintf(
+					'<p><a href="%2$s"><button name="%1$s" id="cc-btn" class="republication-tracker-tool-button page">%1$s</button></a></p>',
```

Comment:
Putting a `<button>` inside an` <a>` is not valid HTML. It is also not working for me in Firefox: when I click it the page just reloads.

Reply:
Changed to have the `<a>` inside the `<button>`.